package.json

```diff
@@ -42,7 +42,7 @@
     "node-uuid": "git+https://github.com/gwicke/node-uuid#master",
     "preq": "~0.3.6",
     "request": "^2.44.0",
-    "restbase-mod-table-cassandra": "^0.5.0",
+    "restbase-mod-table-cassandra": "^0.5.1",
```

Comment:
You'd probably want to make that `^0.5.2` seeing that `^0.5.1` has already been taken by @gwicke :)